openmdao/visualization/n2_viewer/src/N2Toolbar.js

```diff
@@ -261,6 +261,27 @@ class N2Toolbar {
                 self._setRootButton(target);
             });
 
+        new N2ToolbarButtonClick('#hide-connections', tooltipBox,
+            "Remove all connection arrows",
+            function (target) {
+                n2ui.n2Diag.clearArrows();
+                self._setRootButton(target);
+            });
+
+        new N2ToolbarButtonClick('#hide-connections-2', tooltipBox,
```

Comment:
Why are there 2 hide-connections ("hide-connections" and "hide-connections-s")?

Reply:
For toolbar buttons with pullouts, the "primary" button is replaced when a different pullout selection is clicked. E.g., if I hover over the connections button, the pullout appears, and if I click the "Show all connections in view" icon, that button replaces the "hide connections" one at the far left. The icon a button starts with is the same as one of the pullout selections, so they're differentiated by using different ids. 

I just noticed that the tooltip for that button was "Remove all connection arrows" regardless of the whether the button function was hide or show, so I made an update to fix that.

Reply:
Thanks, that clears up my confusion.